packages/cli/generators/app/templates/README.md.ejs

```diff
@@ -0,0 +1,79 @@
+# <%= project.name %>
+
+This application is generated using [LoopBack 4 CLI](https://loopback.io/doc/en/lb4/Command-line-interface.html) with the
+[initial project layout](https://loopback.io/doc/en/lb4/Loopback-application-layout.html).
+
+## Install dependencies
+
+By default, dependencies have been installed when this application was generated.
+Whenever dependencies in `package.json` are changed, run the following command:
+
+```sh
+npm install
+```
+
+To only install resolved dependencies in `package-lock.json`:
+
+```sh
+npm ci
+```
+
+## Run the application
+
+```sh
+npm start
+```
+
+You can also run `node .` to skip the build step.
+
+Open http://127.0.0.1:3000 in your browser.
```

Comment:
Do we want to mention the other http format?

```
http://[::1]:3000
```

Reply:
No, it's safer and simpler to use `127.0.0.1`.

Reply:
ok. It is the default http format i see when I start an lb4 app though.